### DISCUSSION TITLE
No module named 'peft' error

### DISCUSSION THREAD

**Discussion by LCloiseau** [huggingface/peft#2130] [Q&A]
I'm following this tutorial : https://www.youtube.com/watch?v=EaqTEZd6SP4&ab_channel=MyAIForce
But I'm getting a "No module named 'peft'" error on the DownloadAndLoadFlorence2Model node. I have the latest peft installed. My SD is a portable version. Maybe that's the hickup ? (I come from art, not tech, meaning I'm a huge noob xD). How can I get peft inside the portable version ? Or make the node look outside of it ?
Please help ! Thanks ahead.

**Comment by BenjaminBossan** [comment]
It's very hard to say what's going on, I have no idea how Comfy deals with environments. However, this type of error is a very strong indicator that there is an issue with the Python environment. E.g. it could be that the Python environment that you have installed PEFT into is not the same environment that Comfy uses.

Worst case, I'd try to create the whole environment new from scratch.

**Reply by LCloiseau** [reply]
Thank you for your response. This was also what I figured. I'm looking into installing PEFT into my comfyui environment.
